common/djangoapps/philu_commands/management/commands/sync_course_enrollments_with_nodebb.py

```diff
@@ -0,0 +1,43 @@
+from logging import getLogger
+
+from django.core.management.base import BaseCommand
+
+from opaque_keys.edx.keys import CourseKey
+from opaque_keys import InvalidKeyError
+
+from nodebb.helpers import get_community_id
+from nodebb.tasks import task_join_group_on_nodebb
+from student.models import CourseEnrollment
+
+log = getLogger(__name__)
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--course_ids',
+            action='append',
+            help='Sync enrollments of this/these course id(s)',
+        )
+
+    def _get_course_active_enrollments(self, course_id):
```

Comment:
How about adding this method to `CourseEnrollmentManager` or if you want to avoid core change, How about defining this method in helper, As we are also getting active CourseEnrollments in some other commands as well. 
Thoughts?

Reply:
`_get_active_course_enrollments` looks more appropriate to me.

Reply:
How about using proxy models and new Manager to avoid making core changes?

**Note:**
Adding proxy model will generate migrations but doesn't change database schema
**FYI:**
https://stackoverflow.com/questions/37988914/why-does-django-create-migration-files-for-proxy-models

Reply:
We can do this in another iteration later. But good suggestions @zeshanarifios